workflow/rules/align.smk

```diff
@@ -43,7 +43,15 @@ rule Bowtie2_Pathogenome_Alignment:
     benchmark:
         "benchmarks/BOWTIE2/{sample}.benchmark.txt"
     message:
-        "Bowtie2_Pathogenome_Alignment: ALIGNING SAMPLE {input.fastq} TO PATHOGENOME WITH BOWTIE2"
+        "Bowtie2_Alignment: ALIGNING SAMPLE {input.fastq} WITH BOWTIE2"
     shell:
-        """bowtie2 --large-index -x {params.PATHO_DB} --end-to-end --threads {threads} --very-sensitive -U {input.fastq} 2> {log} | samtools view -bS -q 1 -h -@ {threads} - | samtools sort - -@ {threads} -o {output.bam} >> {log};"""
-        """samtools index {output.bam}"""
+        """bowtie2 --large-index -x {params.BOWTIE2_DB} --end-to-end --threads {threads} --very-sensitive -U {input.fastq} > $(dirname {output.bam})/AlignedToBowtie2DB.sam 2> {log};"""
```

Comment:
Since the end result here is a bam file, why not directly write to bam using the `-b` option?

Reply:
Not possible unfortunately because of the duplicate records in the header